### PR TITLE
Release 2.7.0

### PR DESCRIPTION
**Pull request by jdidion**
Fixed some issues with CWL `Any` type conversions.

Most of the file changes are to replace docker image names with dx uris in integration tests.

### CODE REVIEW

File: scripts/run_tests.py [hunk 1]
```diff
@@ -442,7 +442,7 @@
     "importer_ok",
     "read_write_map",
     "docker_image_cache_unspecified",
-    "subworkflow",
+    #"subworkflow",
```
Comment (r-i-v-a) [inline scripts/run_tests.py]: Is this one supposed to be ignored? @jdidion

File: executorCommon/src/main/resources/application.conf [hunk 2]
```diff
@@ -1,5 +1,5 @@
 dxExecutorCommon {
-    version = "2.6.1-SNAPSHOT"
+    version = "2.7.0"
```
Comment (r-i-v-a) [inline executorCommon/src/main/resources/application.conf:2]: Please update working version to 2.7.1-SNAPSHOT for `develop`. Maybe create a new branch starting from `release-2.7.0` and make the PR from that. @jdidion

**Review by r-i-v-a — CHANGES_REQUESTED** [review]
Project working version & ignored test?

File: executorCommon/src/main/resources/application.conf (hunk 2 above)
Comment (jdidion) [reply executorCommon/src/main/resources/application.conf:2]: done

Comment (r-i-v-a) [reply executorCommon/src/main/resources/application.conf:2]: I meant make the other PR https://github.com/dnanexus/dxCompiler/pull/227 (which looks like it has this branch's contents + the version update) into `develop` and replace this PR with that one. @jdidion

**Review by r-i-v-a — CHANGES_REQUESTED** [review]
Use https://github.com/dnanexus/dxCompiler/pull/227 instead for PR to `develop`.

File: executorCommon/src/main/resources/application.conf (hunk 2 above)
Comment (jdidion) [reply executorCommon/src/main/resources/application.conf:2]: let's do them as two separate PRs.

Comment (r-i-v-a) [reply executorCommon/src/main/resources/application.conf:2]: ok.

**Review by r-i-v-a — APPROVED** [review]

**Pull request merged**